aws-codeguru-reviewer.yml

```diff
@@ -0,0 +1,8 @@
+version: 1.0
+
+# Sample YAML file
+# https://docs.aws.amazon.com/codeguru/latest/reviewer-ug/welcome.html.
```

Comment:
I couldn't find any mention of this file format (or suppression in general) in this URL. Is there a better URL?

If not yet published online, where is this format documented in the meantime?

Reply:
will fix